src/services/api/SqliteApi.ts

```diff
@@ -3363,4 +3378,35 @@ order by
       return [];
     }
   }
+  async deleteTeacher(classId: string, teacherId: string) {
+    try {
+      await this.executeQuery(
+        `UPDATE class_user SET is_deleted = 1 WHERE user_id = ? AND class_id = ? AND role = 'teacher'`,
+        [teacherId, classId]
+      );
+
+      const query = `
+      SELECT * 
```

Comment:
Why do we need this query

Reply:
to update the row in supabase while pushing changes